src/components/select/select.js

```diff
@@ -259,7 +260,9 @@ function SelectDirective($mdSelect, $mdUtil, $mdConstant, $mdTheming, $mdAria, $
     }
 
     var isMultiple = $mdUtil.parseAttributeBoolean(attr.multiple);
-
+    //Defaults ','
+    var multipleSeparator = attr.multipleSeparator || ', ';
```

Comment:
I don't think that we need to support an infinite list of options here. Plus doing so seems to expose us to XSS (see below).

There is no guidance from [the MD spec](https://material.io/design/components/text-fields.html#anatomy) on this currently.

Let's create a finite list of options that can be passed into the attribute like the following

- comma (default)
- semi-colon
- `|` whatever that is called...
- space
- plus

I'm open to expanding this list if any other ideas come to mind. However, it won't be too hard to add to it later if we need to.

That should probably be called `selection-separator-char`.

Then for `selection-separator-spacing`, we should probably allow the following

- after (default)
- before
- none
- break - where this would ignore the `selection-separator-char`

I think that these 2 sets of options give us a ton of flexibility without causing us too much of a maintenance burden or exposing us to XSS.